docker-compose-pro.yml

```diff
@@ -7,14 +7,12 @@ services:
     ports:
       - "127.0.0.1:4566:4566"            # LocalStack Gateway
       - "127.0.0.1:4510-4559:4510-4559"  # external services port range
-      - "127.0.0.1:53:53"                # DNS config (required for Pro)
-      - "127.0.0.1:53:53/udp"            # DNS config (required for Pro)
       - "127.0.0.1:443:443"              # LocalStack HTTPS Gateway (required for Pro)
     environment:
+      # LocalStack configuration: https://docs.localstack.cloud/references/configuration
       - DEBUG=${DEBUG-}
       - PERSISTENCE=${PERSISTENCE-}
```

Comment:
Do we want to keep the persistence configuration in the official docker-compose-pro.yml?

Reply:
i think this is a good way for us to advertise a pro feature, even though i'm on the fence about forwarding environment variables in this way.

anyway i tend towards leaving it for now